### DISCUSSION TITLE
✅ Add multiple ports to tunnel configuration

### DISCUSSION THREAD

**Discussion by bytefactory73** [isontheline/pro.webssh.net#219] [Ideas]
I wish to add multiple ports in the tunnel configuration. 
eg:
8020:gitlab.mycompany.com:8020
8030:buildserver.mycompany.com:8030

this is how many other tools work and what i find a bit limiting in webssh.

thx
tilo

**Comment by isontheline** [comment]
Hello @bytefactory73 👋

Good idea!

I have created an issue to track progress : https://github.com/isontheline/pro.webssh.net/issues/220

**Reply by isontheline** [reply]
Hello @bytefactory73 👋

Could you try 17.3 beta release?
https://testflight.apple.com/join/QSrBK59z
